src/main/kotlin/org/rust/ide/template/postfix/MatchPostfixTemplate.kt

```diff
@@ -45,8 +47,27 @@ class MatchPostfixTemplate(provider: RsPostfixTemplateProvider) :
 
         PsiDocumentManager.getInstance(project).doPostponedOperationsAndUnblockDocument(editor.document)
 
-        val arm = matchExpr.matchBody?.matchArmList?.firstOrNull() ?: return
-        val blockExpr = arm.expr as? RsBlockExpr ?: return
-        editor.caretModel.moveToOffset(blockExpr.block.lbrace.textOffset + 1)
+        val matchBody = matchExpr.matchBody ?: return
+        val wildPatterns = collectWildPatterns(matchBody)
```

Comment:
Why not just `descendantsOfType<RsPatWild>()`? Seems the same but simpler

Reply:
Ah, of course :) Thanks.